src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/MissingOverride/context/defaultContext.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/Test.java b/Test.java
+index 0999e31..6310d55 100644
+--- a/Test.java
++++ b/Test.java
+@@ -3,19 +3,21 @@ package TreeWalker.MissingOverride;
+ public class Test {
+ 
+     /** {@inheritDoc} */
+-    @Override
```

Comment:
First Problem: There is only a deleted line, but this deleted line will cause a violation in MissingOverride: Test.java:6:5: Must include @java.lang.Override annotation when {@inheritDoc} Javadoc tag exists.
but now in current suppressionPatchXpathFilter version, we only consider the added/changed line, not deleted line. The reason that deleted line does not be considered is that deleted line does not belong to current java file that do checkstyle, so it is difficult to deal with it. So how do solve this situation?

Reply:
This is huge problem .... as removed code does not exists during checkstyle execution we cannot bind it to context at ALL.
We might have it as limitation by design and just suppress violation.

It is better to suppress extra violation, than do false-positive. PatchFilter is transitional Filter to be used for some time till project become ready to use more strict filters.

Reply:
is it related to this issue https://github.com/checkstyle/patch-filters/issues/160 ?

Reply:
yes, this related to #160, but difference is that this deleted line is not belong to violations ast's child node.